package.json

```
@@ -40,10 +40,9 @@
     "body-parser": "1.18.3",
     "dateformat": "3.0.3",
     "express": "~4.16.4",
-    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
+    "iotagent-node-lib": "git://github.com/jason-fox/iotagent-node-lib.git#feature/request-shim",
```

As described in https://github.com/telefonicaid/iotagent-node-lib/pull/1119#issuecomment-1000483835, this need to be addressed (reverted) as first step of the process. 

```suggestion
    "iotagent-node-lib": "git://github.com/telefonicaid/iotagent-node-lib.git#master",
```